### PR TITLE
Vb askfortelephone

### PR DESCRIPTION
**Pull request by veltsonbastien**
All that's left should be the fixing up in the backend.

### CODE REVIEW

File: src/layers/identity.tsx [hunk 1]
```diff
@@ -78,6 +81,7 @@ export class Identity {
     if (response.status === 200) {
       this.name = json.name;
       this.email = json.email;
+      this.phone = json.phone;
```
Comment (alex-shortt) [inline src/layers/identity.tsx:84]: don't see a reason to return the phone number from the backend user_info function

File: src/layers/identity.tsx [hunk 2]
```diff
@@ -94,6 +98,7 @@ export class Identity {
     this.setToken(undefined);
     this.name = undefined;
     this.email = undefined;
+    this.phone = undefined;
```
Comment (alex-shortt) [inline src/layers/identity.tsx:101]: don't see a reason to return the phone number from the backend user_info function

File: src/worlds/MuseHQ/dialogue/signup.ts [hunk 3]
```diff
@@ -56,13 +57,22 @@ export const useSignupLogic = (inKey: string, outKey: string): Dialogue => {
...
         { name: "go back", nextKey: "signup-password" },
       ],
     },
+    {
+      key: "signup-phone",
+      text: "what's your phone number? (optional)",
+      input: { value: phone, setValue: setPhone, type: "text", persist: false },
+      decisions: [
+        { name: "next", nextKey: "signup-artist" },
+        { name: "cancel", nextKey: inKey },
```
Comment (alex-shortt) [inline src/worlds/MuseHQ/dialogue/signup.ts:73]: would personally stray away from punctuation like that and keep it conversational. thinking..

"what's your phone number?"

but then add a skip option

{ name: "submit", nextKey: "signup-artist" },
{ name: "skip", nextKey: "signup-artist" }

**Review by alex-shortt — CHANGES_REQUESTED** [review]
also on the backend side, i should run you through how to set up a local environment for it. i can do this one now, but let's find a time this week

File: src/layers/identity.tsx (hunk 1 above)
Comment (veltsonbastien) [reply src/layers/identity.tsx:84]: done 👍

File: src/layers/identity.tsx (hunk 2 above)
Comment (veltsonbastien) [reply src/layers/identity.tsx:101]: done 👍

File: src/worlds/MuseHQ/dialogue/signup.ts (hunk 3 above)
Comment (veltsonbastien) [reply src/worlds/MuseHQ/dialogue/signup.ts:73]: done 👍

**Review by alex-shortt — APPROVED** [review]

**Pull request merged**